diktat-rules/src/test/kotlin/org/cqfn/diktat/util/DiktatRuleSetProvider4Test.kt

```diff
@@ -17,7 +17,7 @@ import kotlin.reflect.jvm.reflect
  */
 class DiktatRuleSetProvider4Test(private val ruleSupplier: (rulesConfigList: List<RulesConfig>) -> Rule,
                                  rulesConfigList: List<RulesConfig>?) : RuleSetProvider {
-    private val rulesConfigList: List<RulesConfig>? = rulesConfigList ?: RulesConfigReader(javaClass.classLoader).readResource("rules-config.json")
+    private val rulesConfigList: List<RulesConfig>? = rulesConfigList ?: RulesConfigReader(javaClass.classLoader).readResource("../diktat-analysis.yml")
```

Comment:
```suggestion
    private val rulesConfigList: List<RulesConfig>? = rulesConfigList ?: RulesConfigReader(javaClass.classLoader).readResource("diktat-analysis.yml")
```
I believe there is no need to change path to config for tests too: it we need to override specific settings, we do it in code, and otherwise example `diktat-analysis` will work fine.